main.go

```diff
@@ -366,8 +417,12 @@ func main() {
 
 	// send events to Caduceus using vegeta
 	var metrics vegeta.Metrics
+	currentTime := time.Now()
+	currentFunc := func() time.Time {
+		return currentTime
+	}
```

Comment:
if the time function always returns the time from startup, why not just pass that time.Time value?

Reply:
Yeah that would be the better move